### ISSUE TITLE
Q: How to enable Portuguese? 

### ISSUE THREAD

**Issue by FranciscoCaldeira** [streetsidesoftware/vscode-spell-checker#3334]
### Version

v4.0.1

### OS

Windows

### Description

have this in extension ` "cSpell.language": "en-US,en-GB,pt_PT" `

but string 'super administrador' allerts:
![image](https://github.com/streetsidesoftware/cspell/assets/45494986/a1ef9a7f-6d72-455e-b207-fddf53f10f31)

### Steps to Reproduce

add language "cSpell.language": "en-US,en-GB,pt_PT"

write en and pt words.

### Expected Behavior

don't allert

### cspell.json

```jsonc
-
```

### cspell.config.yaml

```yml
-
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Jason3S** [comment]
@FranciscoCaldeira,

Do you have the extension: [Portuguese - Code Spell Checker - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-portuguese) installed?

**Comment by Jason3S** [comment]
@FranciscoCaldeira,

The spell checker does not come with Portuguese by default, you have to install it: [Portuguese - Code Spell Checker - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-portuguese).

**Comment by Jason3S** [comment]
> I later installed the ext your suggested, but some words don't work like: `administracao` (incorrect) -> `administração` (correct)

What are you hoping for?
![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/fa4d7c9c-4978-4864-a57e-ee8e7f90f792)

Are you expecting it to ignore the accents or do you want it to check the accents?

To turn off accents is a bit more complicated. You have to use `cSpell.languageSettings`

![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/2e58ae8f-e61f-4814-a1b2-65e7d0c2fe40)


**`.vscode/settings.json`**

```jsonc
{
    "cSpell.language": "en,pt",
    "cSpell.caseSensitive": false,
    "cSpell.languageSettings": [
        {
            "locale": "pt,pt-PT",
            "languageId": "markdown,python,typescript,javascript",
            "caseSensitive": false
        }
    ]
}
```

![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/3740137/706e1a8f-422b-40f2-8e2b-80c51af55474)

**Comment by FranciscoCaldeira** [comment]
thanks, @Jason3S for all the tips/help. 

> Are you expecting it to ignore the accents or do you want it to check the accents?

I expected it to check the accents, the word doesn't warn while it's wrong written.

See:
![image](https://github.com/streetsidesoftware/vscode-spell-checker/assets/45494986/d3e45bf3-0db2-4975-8c22-6c70b495dd13)

**Comment by Jason3S** [comment]
@FranciscoCaldeira,

Please try:

**`.vscode/settings.json`**

```jsonc
{
    "cSpell.language": "en-us,en-gb,pt",
    "cSpell.caseSensitive": true,
    "cSpell.languageSettings": [
        {
            "locale": "pt,pt-PT",
            "languageId": "*",
            "caseSensitive": true
        }
    ]
}
```

**Comment by FranciscoCaldeira** [comment]
thanks, 👍
